share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md

```diff
@@ -0,0 +1,51 @@
+# How to Create and Maintain a Tap
+Taps are external sources of Homebrew formulae. They can be created by anyone
```

Comment:
@mikemcquaid This is mentioned at the end: https://github.com/Homebrew/homebrew/pull/41858#discussion_r35016491